server/src/connectionHandlers/onCompletion.ts

```diff
@@ -109,7 +109,40 @@ export function onCompletionHandler (textDocumentPositionParams: TextDocumentPos
     }
   }
 
-  let symbolCompletionItems: CompletionItem[] = []
+  const symbolCompletionItems: CompletionItem[] = getSymbolCompletionItems(word)
```

Comment:
this function is >100 lines long. I think it's easily divisible into smaller ones.

Reply:
I the new version I cut the function in order to distinguish better bash, python and bitbake regions. However the remainder of the function is not my code, and I'd rather we don't start refactoring each other code over subjective preferences. This could lead to chaos 🙃